modules/couchbase-cluster/main.tf

```diff
@@ -1,33 +1,45 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# REQUIRE A SPECIFIC TERRAFORM VERSION OR HIGHER
+# This module has been updated with 0.12 syntax, which means it is no longer compatible with any versions below 0.12.
+# ----------------------------------------------------------------------------------------------------------------------
+
 terraform {
-  required_version = ">= 0.10.0"
+  required_version = ">= 0.12"
 }
 
 # ---------------------------------------------------------------------------------------------------------------------
 # CREATE AN AUTO SCALING GROUP (ASG) TO RUN COUCHBASE AND/OR SYNC GATEWAY
 # ---------------------------------------------------------------------------------------------------------------------
 
 resource "aws_autoscaling_group" "autoscaling_group" {
-  name = "${var.cluster_name}"
-
-  launch_configuration = "${aws_launch_configuration.launch_configuration.name}"
-  vpc_zone_identifier  = ["${var.subnet_ids}"]
-
-  min_size             = "${var.min_size}"
-  max_size             = "${var.max_size}"
-  termination_policies = ["${var.termination_policies}"]
-
-  health_check_type         = "${var.health_check_type}"
-  health_check_grace_period = "${var.health_check_grace_period}"
-  wait_for_capacity_timeout = "${var.wait_for_capacity_timeout}"
-
-  tags = [
-    {
-      key                 = "Name"
-      value               = "${var.cluster_name}"
-      propagate_at_launch = true
-    },
-    "${var.tags}",
-  ]
+  name = var.cluster_name
+
+  launch_configuration = aws_launch_configuration.launch_configuration.name
+  vpc_zone_identifier  = var.subnet_ids
+
+  min_size             = var.min_size
+  max_size             = var.max_size
+  termination_policies = [var.termination_policies]
```

Comment:
This was confusing to me because I thought `termination_policies` was a list but it is actually a string input. Can you add a comment here indicating that?

Reply:
Hm, I think you're right, it should be a list. https://github.com/gruntwork-io/terraform-aws-couchbase/pull/49/commits/31d9a70aaba2d4546225efd1c0c023e1931f0304